code/AssetLib/3DS/3DSConverter.cpp

```diff
@@ -686,8 +686,8 @@ void Discreet3DSImporter::GenerateNodeGraph(aiScene *pcOut) {
     pcOut->mRootNode = new aiNode();
     if (0 == mRootNode->mChildren.size()) {
         //////////////////////////////////////////////////////////////////////////////
-        // It seems the file is so messed up that it has not even a hierarchy.
-        // generate a flat hiearachy which looks like this:
+        // It seems the file is so messed up that it does not even have a hierarchy.
+        // Generate a flat hiearachy which looks like this:
```

Comment:
hiearachy -> hierarchy
(Should fix the spelling errors too)

Reply:
I'll keep this in mind. Currently trying to work on some other stuff.